static/js/settings.coffee

```diff
@@ -208,7 +208,7 @@ $().ready ->
         ($ "#request-error .alert").show()
         ($ "#request-error .alert").addClass "alert-success"
         ($ "#request-error .alert").removeClass "alert-danger"
-        ($ "#request-error .msg").text "Screenly shutdown has started successfully."
+        ($ "#request-error .msg").text "Screenly shutdown has started successfully. Soon you will be able to turn off the device manually."
```

Comment:
"Soon you will be able to turn off the device manually." -> "Soon you will be able to unplug the power from your Raspberry Pi."